### PR TITLE
Added refactoring of test Creation Multiconfiguration Project 23.11.23

### PR DESCRIPTION
**Pull request by Tarantata**
https://trello.com/c/L4S4bFTV/459-rf-pom-tc0306008

### CODE REVIEW

File: cypress/e2e/pom_e2e/mcProject.cy.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import McProjectHomePage from "../../pageObjects/McProjectHomePage";
+import mcProjectData from "../../fixtures/pom_fixtures/mcProjectData.json"; 
+// import McProjectNewJobPage from "../../pageObjects/McProjectNewJobPage";
+// import McProjectConfigProjectPage from "../../pageObjects/McProjectConfigProjectPage";
+
+describe('mcProject', () => {    
+
+    const mcProjectHomePage = new McProjectHomePage;
+    // const mcProjectNewJobPage = new McProjectNewJobPage;
+    // const mcProjectConfigProjectPage = new McProjectConfigProjectPage;
```
Comment (Rom-T) [inline cypress/e2e/pom_e2e/mcProject.cy.js:11]: Please delete comments

File: cypress/e2e/pom_e2e/mcProject.cy.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/// <reference types="cypress"/>
+import McProjectHomePage from "../../pageObjects/McProjectHomePage";
+import mcProjectData from "../../fixtures/pom_fixtures/mcProjectData.json"; 
+// import McProjectNewJobPage from "../../pageObjects/McProjectNewJobPage";
+// import McProjectConfigProjectPage from "../../pageObjects/McProjectConfigProjectPage";
```
Comment (Rom-T) [inline cypress/e2e/pom_e2e/mcProject.cy.js:5]: Please delete comments

File: cypress/e2e/pom_e2e/mcProject.cy.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+describe('mcProject', () => {    
+
+    const mcProjectHomePage = new McProjectHomePage;
+    // const mcProjectNewJobPage = new McProjectNewJobPage;
+    // const mcProjectConfigProjectPage = new McProjectConfigProjectPage;
+
+    it('Create Multiconfiguration project', function() {
```
Comment (S-Lanchik) [inline cypress/e2e/pom_e2e/mcProject.cy.js:13]: Add number of your test

**Comment by Tarantata** [comment]
There is a lot of faults, so, I cancel it.

**Pull request closed without merging**